packages/expo/src/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { View, Alert, Button as SignInButton } from "react-native";
+
+const Button = () => {
+    return (
+      <View className="flex flex-col space-y-2">
+       <SignInButton
+          title="button"
+          onPress={() => Alert.alert('button pressed')}
+        />
+      </View>
```

Comment:
The view is not necessary
```suggestion
       <SignInButton
          title="button"
          onPress={() => Alert.alert('button pressed')}
        />
```

Reply:
ok